### PR TITLE
(grc_conversion) EpiDoc and CTS conversion

### PR DESCRIPTION
**Pull request by lcerrato**
work continues on batch #1399

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu I have a few more things in the queue but I don't want to overload the PRs because the biblio stuff is confusing.

File: data/tlg0199/tlg001/tlg0199.tlg001.perseus-eng2.xml
Comment (AlisonBabeu) [inline data/tlg0199/tlg001/tlg0199.tlg001.perseus-eng2.xml:1]: Could delete ` <biblScope unit="volume"/>`

**Review by AlisonBabeu — APPROVED** [review]
Looks good!

File: data/tlg0199/tlg001/tlg0199.tlg001.perseus-eng2.xml
Comment (lcerrato) [reply data/tlg0199/tlg001/tlg0199.tlg001.perseus-eng2.xml:1]: Just placeholders so that things aren't omitted.

**Pull request merged**